debian/changelog

```diff
@@ -1,3 +1,60 @@
+ubuntu-advantage-tools (27.1) impish; urgency=medium
+
+  * d/control:
+    - specify debianutils min version
+  * lintian:
+    - override xenial specific errors
+    - rename package-specific overrides for pro vs tools
+    - override ubuntu-advantage-pro wanted-by-target cloud-init
+  * New upstream release 27.1:
+    - changelog: fix lintian typos amend and redact incorrect 27.0 entry
+      (GH: #1624)
+    - apt-hook:
+      + avoid segfault when comparing null Apt file origin to esm
+        (GH: #1929123)
+      + fix lintian warnings based on compiled binary (GH: #1626)
+      + only add newlines for MOTD if message file length is non-zero
+      + avoid wrapping static message formats at 80 chars
+    - docs: update releases.md per SRU review feedback on branch structuring
+    - cis:
+      + make service non-beta
+      + update cis help url
+      + post enable message pointing to docs
+    - fips:
+      + Block enabling fips if fips-updates once enabled (#1623) (GH: #1600)
+      + clarify messaging (GH: #1631)
+    - enable: correct messaging for beta service (GH: #1588)
+    - errors: print a more helpful message when ssl fails (GH: #1618)
+    - pro: obtain AWS IMDSv2 API token before trying to grab pkcs7 doc
+      (GH: #1608)
+    - livepatch: alert when snapd does not have wait cmd (GH: #1927905)
+    - messaging:
+      + esm announcement message reports u.com/16-04
+      + separate _remove_msg_template. emit no warranty on infra disabled
+      + point to u.com/16-04 instead of u.com/advantage on ESM (GH: #1584)
+      + Infra and Apps messaging is mutually exclusive (GH: #1573)
+    - Do not show beta services in autocomplete (GH: #1594)
+    - logging: remove tracebacks for UserFacingErrors (GH: #1586)
+    - attach: do not print contract name if empty
+    - status: do not show info if not on contract (GH: #1592)
+    - tests
```

Comment:
I think we can remove some of those messages. We should keep in the changelog changes that the affect that are meaningful to the user. I know this is a little bit subjective, but entries like:

```
fix tox for new behave tags
add ability to inject custom userdata to a CI run
```

Don't seem meaningful to the user in my opinion and we can remove them

Reply:
I think we are missing a `:` here